system/lib/fetch/asmfs.cpp

```diff
@@ -975,9 +975,7 @@ static long open(const char* pathname, int flags, int mode) {
       // cannot stop to wait for it to finish, and must return a failure (file not found)
       if (emscripten_is_main_browser_thread()) {
         if (emscripten_fetch_wait(node->fetch, 0) != EMSCRIPTEN_RESULT_SUCCESS) {
-          RETURN_ERRNO(ENOENT, "Attempted to open a file that is still downloading on the main "
-                               "browser thread. Could not block to wait! (try preloading the file "
-                               "to the filesystem before application start)");
+          RETURN_ERRNO(ENOENT, "Attempted to open a file that is still downloading on the main browser thread. Could not block to wait! (try preloading the file to the filesystem before application start)");
```

Comment:
This seems strictly less-good as a single long line to me. I like having multiple tabs open on one monitor, sometimes I've only got a laptop, etc.

Reply:
I agree it's less readable, yeah. But this does at least unbreak that code, so I think it's worth committing - the code as it is in the repo just doesn't build.

Reply:
How is it possible this code simply doesn't build?  How did it get passed CI?

If its existing broken code then can we fix it in separate CL?

Reply:
It's behind a debug flag, which is off by default, and must be manually enabled in the code.

I'll move it to another PR.